internal/metamorphic/options.go

```diff
@@ -348,6 +348,10 @@ func randomOptions(rng *rand.Rand) *testOptions {
 		opts.Experimental.MaxWriterConcurrency = 2
 		opts.Experimental.ForceWriterParallelism = true
 	}
+	if rng.Intn(2) == 0 {
+		// Disable this for half the random options.
+		opts.Experimental.IngestSSTablesAsFlushable = false
```

Comment:
Instead of having this rely on the default, it'd be more interesting to just unconditionally set:

```
opts.Experimental.IngestSSTablesAsFlushable = rng.Intn(2) == 0
```